rts/at/sra/README.md

```diff
@@ -0,0 +1,68 @@
+OVERVIEW
+--------
+An example of how to use REST-AT service.
+
+USAGE
+--------------------
+TEST SRA PARTICIPANTS
+
+Start a REST-AT coordinator on port 8080
+
+    cd <narayana-repo>/jboss-as/build/target/wildfly-28.0.0.Final
```

Comment:
The parent README for REST-AT already describes how to run the coordinator, can you add a markdown link to the relevant section instead of duplicating the text because the risk is that they may diverge. Doing it this way also has the benefit of keeping the README focused on just the example.

Reply:
This is updated, so resolving the conversation.